views/pages/services/student-services.pug

```diff
@@ -0,0 +1,52 @@
+extends /views/templates/frontend-base.pug
```

Comment:
could you remove all the copy from this page and put it into a data file in the `server/data` dir? that way the content on this page can be updated through the cms, instead of having to come into this file and change pug markdown